tests/masonry/utils/test_tags.py

```diff
@@ -0,0 +1,23 @@
+from poetry.masonry.utils.tags import get_abi_tag
+from poetry.utils.env import MockEnv
+
+
+def test_tags_cpython38():
```

Comment:
Would be great to add a `< 3.8` case as well here.

Reply:
There now is a test for CPython 3.7 as well.